autohooks/config.py

```diff
@@ -84,11 +84,11 @@ def is_empty(self) -> bool:
         return not bool(self._config_dict)
 
 
-def _gather_mode(mode: Optional[str]) -> Mode:
+def _gather_mode(mode_string: Optional[str]) -> Mode:
     """
     Gather the mode from a mode string
     """
-    mode = Mode.from_string(mode)
+    mode = Mode.from_string(mode_string)
```

Comment:
Not sure why this change is necessary. Personally I don't like adding a type to a variable name. Would like to revert this change.

Reply:
this is necessary because mode is defined as an Optional[str] but later overwriten with a "Mode"-Object. That's a problem for mypy. The name is inspired by the parameter of "from_string", that is also called modestring

Reply:
Do you have a cleaner solution in mind? In my opinion that's the best way to go (including the variable name)

Reply:
Ok should be fine then :-)